web-www/src/components/ConnectSection.tsx

```diff
@@ -78,17 +78,19 @@ const IconLabel = ({
         <Box
           sx={theme => ({
             bgcolor: 'secondary.main',
+            background:
+              'linear-gradient(201.8deg, #4FB573 14.67%, #B9E1C7 97.14%);',
             borderRadius: '50%',
             transitionDuration: '200ms',
             transitionProperty: 'color, background-color',
             transitionTimingFunction: 'ease-in-out',
             width: {
               xs: theme.spacing(15),
-              tablet: theme.spacing(30),
+              tablet: 80,
```

Comment:
We can't use the theme.spacing util anymore?

Reply:
actually an oversight - good catch